crates/vcs_menu/src/lib.rs

```diff
@@ -131,6 +131,43 @@ impl BranchListDelegate {
             model.show_toast(Toast::new(GIT_CHECKOUT_FAILURE_ID, message), ctx)
         });
     }
+
+    fn create_branch(cx: &mut ViewContext<Picker<Self>>) {
+        cx.spawn(|picker, mut cx| async move {
+                                picker.update(&mut cx, |this, cx| {
+                                    let project = this.delegate.workspace.read(cx).project().read(cx);
+                                    let current_pick = &this.delegate.last_query;
+                                    let mut cwd = project
+                                    .visible_worktrees(cx)
+                                    .next()
+                                    .ok_or_else(|| anyhow!("There are no visisible worktrees."))?
+                                    .read(cx)
+                                    .abs_path()
+                                    .to_path_buf();
+                                    cwd.push(".git");
+                                    let repo = project
+                                        .fs()
+                                        .open_repo(&cwd)
+                                        .ok_or_else(|| anyhow!("Could not open repository at path `{}`", cwd.as_os_str().to_string_lossy()))?;
```

Comment:
I know you just copied the code (that's fine!), but: since you're using `visible_worktrees(cx).next()` here this will break if you have multiple worktree. If you want to fix that, can you try the approach here? https://github.com/zed-industries/zed/pull/8888

Reply:
Thank you, for pointing that out. I didn't realise that this could lead to issues. I will look into it.